packages/daimo-api/src/db/db.ts

```diff
@@ -131,6 +131,11 @@ export class DB {
           ALTER TABLE linked_account ADD COLUMN IF NOT EXISTS created_at TIMESTAMP DEFAULT NOW();
           ALTER TABLE used_faucet_attestations ADD COLUMN IF NOT EXISTS created_at TIMESTAMP DEFAULT NOW();
 
+          CREATE TABLE IF NOT EXISTS session_key (
+            secret VARCHAR(184) PRIMARY KEY,
+            pubkey VARCHAR(184) NOT NULL, -- DER-encoded Hex P256-SHA256 public key
```

Comment:
maybe worth storing which account created the key. we can look it up by pubkey but there are a few instances of (1 pubkey > more than one daimo account)